crates/swc/tests/fixture/issues-2xxx/2964/case-2/output/index.ts

```diff
@@ -10,8 +10,8 @@ function _export(target, all) {
 }
 _export(exports, {
     badIstanbul: ()=>badIstanbul,
```

Comment:
https://github.com/swc-project/swc/blob/428ca6a650cc221d017245568b71e43c7c440fca/crates/swc/tests/fixture/issues-2xxx/2964/case-2/input/index.ts#L1-L8

badIstanbul's span is lost. 

This issue will disappear if `...pixelParams` is removed.
I supposed it is a bug related to `object_rest_spread`.

Reply:
I will fix this soon in another pull request.

Reply:
Cool, thanks!